src/components/Card/Card.js

```diff
@@ -27,11 +37,8 @@ function Card(props) {
   };
 
   const clickSaveButtonHandler = () => {
-    console.log(props.id); // Убедитесь, что это значение определено
     setIsEditing(!isEditing);
-    props.handleCatChange(props.id, editedTitle, editedDescr);
-    console.log('---');
-    console.log(props.id, editedTitle, editedDescr);
+    props.handleLoremChange(props.id, editedTitle, editedDescr);
```

Comment:
Could you please name all the 'lorem' variables in more general way?
Here, for example, could be any data we would want, not only Lorem Ipsum text :)